### PR TITLE
Add pixiv.desktop

### PR DESCRIPTION
**Pull request by Thesola10**
This is a WIP towards #4 and the Exec field is subject to change.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 os.system('mkdir -p ~/.local/share/koneko')
 os.system('cp -r ./pics/ ~/.local/share/koneko/')
 os.system('cp example_config.ini ~/.local/share/koneko/')
+os.system('cp pixiv-url.desktop ~/.local/share/koneko/')
```
Comment (akazukin5151) [inline setup.py:15]: Don't you mean `~/.local/share/applications/` here?

Comment (Thesola10) [reply setup.py:15]: I don't want to install it directly there. Say a user who already has a config+token reinstalls `koneko`, we wouldn't want them to end up with a useless `.desktop` file potentially conflicting with another app. I think `koneko` should copy the file to `~/.local/share/applications` when it is run.

**Comment by akazukin5151** [comment]
I'm going to merge this PR before working on `koneko url-login`, if you don't have any pending commits. I'll make a PR later to merge this branch onto master, so you can review it.

As an aside, `koneko` uses docopt to parse arguments, but it assumes all commands are user-facing and publicly documented, which isn't the case here. So it's easier to use a separate `koneko-url-login` program. `lscat` is already a separate program anyway so I think it's better to separate things. There is also little code in common. Thoughts?

**Comment by Thesola10** [comment]
Making a separate `koneko-url-login` is perfectly fine by me, merge away!

**Pull request merged**